sysutils/gptfdisk/Portfile

```diff
@@ -0,0 +1,39 @@
+# $Id:  $
+
+PortSystem              1.0
+
+name                    gptfdisk
+version                 1.0.1
+categories              sysutils
+platforms               darwin
+license                 GPL-2+
+maintainers             mail.ru:nano103
+description             GUID partition table fdisk
+long_description        GPT fdisk is a disk partitioning tool loosely modeled on Linux fdisk, \
+                        but used for modifying GUID Partition Table (GPT) disks. \
+                        The related FixParts utility fixes some common problems \
+                        on Master Boot Record (MBR) disks.
+
+homepage                http://www.rodsbooks.com/gdisk/
+master_sites            sourceforge:project/gptfdisk/gptfdisk/${version}
+
+
+checksums               md5     d7f3d306b083123bcc6f5941efade586 \
+                        sha1    ad28c511c642235815b83fffddf728c117057cba \
+                        sha256 864c8aee2efdda50346804d7e6230407d5f42a8ae754df70404dd8b2fdfaeac7 \
+                        rmd160 21bef329361690a1a7d44e464eb9d682c6335137
+
+depends_lib             port:ncurses \
+                        port:popt
+
+patchfiles              Makefile.mac-install.diff
+post-patch {
+    reinplace "s|__PREFIX__|${prefix}|g" ${worksrcpath}/Makefile.mac-install
+    reinplace "s/-O2/-O2 ${configure.cc_archflags}/g" ${worksrcpath}/Makefile.mac
```

Comment:
Will `${configure.cc_archflags}` already be expanded at this point? Ports usually pass flags via
```
build.args    ... \
              CC="${configure.cc} [get_canonical_archflags]"
```